src/js/control-bar/text-track-controls/chapters-button.js

```diff
@@ -105,7 +97,17 @@ class ChaptersButton extends TextTrackButton {
       }));
     }
 
-    if (chaptersTrack) {
+    if (chaptersTrack && chaptersTrack.cues == null) {
+      chaptersTrack['mode'] = 'hidden';
+
+      let remoteTextTrackEl = this.player_.remoteTextTrackEls().getTrackElementByTrack_(chaptersTrack);
```

Comment:
looks like there an issue here tracks that were added in the html:
```html
  <video id="vid1" class="video-js vjs-default-skin" controls preload="auto" width="640" height="264"
      poster="http://video-js.zencoder.com/oceans-clip.png">
    <source src="oceans-clip.mp4" type='video/mp4'>
    <source src="http://video-js.zencoder.com/oceans-clip.mp4?fo23" type='video/mp4'>
    <track kind="chapters" src="../docs/examples/shared/example-captions.vtt" srclang="en" label="Subs"></track>
</video>
```
What happens here is that we get the chapters track but when we try to retrieve its associated track el, we get undefined so the chapters button never loads.
I think what we need to do is in [`proxyNativeTextTracks_`](https://github.com/videojs/video.js/blob/a601fbb83fdc442f30f09ca15ff55b04c3e7c91d/src/js/tech/html5.js#L254) to loop through any track elements and add them to the remoteTextTrackEls list. And then in [`addRemoteTextTrack`](https://github.com/videojs/video.js/blob/a601fbb83fdc442f30f09ca15ff55b04c3e7c91d/src/js/tech/html5.js#L760) we need to make sure the track el gets added to the list. We also want to make sure that `remoteRemoteTextTrack` removes the associated el.
`html-track-element-list` might need to get updated to support this but hopefully it'll just work.